workflows/phylogenetics/wf_snippy_tree.wdl

```diff
@@ -104,10 +111,10 @@ workflow snippy_tree_wf {
     File? snippy_tree_gubbins_timetree = gubbins.gubbins_timetree
     File? snippy_tree_gubbins_timetree_stats = gubbins.gubbins_timetree_stats
     # snpdists outputs
-    String snippy_tree_snpdists_version = select_first([snp_dists_gubbins.version, snp_dists_iqtree.version])
+    String snippy_tree_snpdists_version = snp_dists_iqtree.version
     # reorder matrix outputs
-    File? snippy_tree_gubbins_matrix = reorder_matrix_gubbins.ordered_matrix
-    File? snippy_tree_gubbins_tree = reorder_matrix_gubbins.tree
+    File? snippy_tree_gubbins_matrix = reorder_matrix_iqtree.ordered_matrix
```

Comment:
And you can remove the `_gubbins_` versions since they will be identical to the `_iqtree_` versions of these outputs